docs/flow-support.md

```diff
@@ -1,69 +1,27 @@
 # Flow Support
 
-Styled-components has first-level [flow](https://flowtype.org) support to help
-you find typing errors while using our public API.
+To use flowtype with the public api of styled-components we recommend that
+you use the library definition in [flow-typed].
 
-Sadly, `flow` does not work right out-of-the-box, since there isn't a
-best practise how to isolate typing dependencies etc. on a module level yet.
+To install it you can use the flow-typed cli or download it manually from
+the git repository and storing it in a flow-typed folder in the same folder
+as your flowconfig.
 
-This document should give you an idea how to setup your `.flowconfig`, so you
-can use `styled-components` without any hassle.
+### Installing the libdef with flow-typed
 
-(First we will give you some context what `flow` requires to type-check
- `node_modules/styled-components` properly,... if you are not interested in the
- details, just go to the `.flowconfig` example at the end of this document)
-
-## Libdef Dependencies
-
-A libdef is a descriptive file for an external untyped 3rd party module used by
-our library. You can find all our dependencies in the
-[flow-typed](../flow-typed) directory.  All files located in
-[flow-typed/npm](../flow-typed/npm) are downloaded or auto-stubbed versions via the `flow-typed` binary,
-files located in `flow-typed/` are adapted versions of `flow-typed/npm` files.
-
-Those adapted files might collide with your locally used libdefs like
-`lodash`... while adapting some libdefs, we made sure not to introduce breaking
-changes, so you can safely use them instead of the official ones found on
-`flow-typed`.
-
-* `flow-typed/lodash_v4.x.x.js` : We added some declarations for file-specific imports (e.g. `lodash/isFunction`)
-* `flow-typed/react-native.js` : This will be very useful for non-react-native related applications, to mute react-native related import errors
-
-It has to be noted that some of these libdef files are mostly "stubs", so they
-default to type `any`, which is kinda bad. We would really appreciate help in
-contributing libdef typings to the [`flow-typed`](https://github.com/flowtype/flow-typed)
-project, to make this project even more typesafe!
+```shell
+npm i -g flow-typed # if you do not already have flow-typed
+flow-typed install styled-components@<version>
```

Comment:
is `-f <flow-bin-version>` still necessary? I thought flow-typed detected it and gave you a friendly error message on how to specify if it can't be detected.